lib/rucio/web/rest/flaskapi/v1/redirect.py

```diff
@@ -283,17 +274,7 @@ def get(self, scope_name):
                             else:
                                 return 'no redirection possible - no valid RSE for HTTP redirection found', 404, headers
                         else:
-                            if select == 'geoip':
-                                rep = sort_geoip(dictreplica, client_location['ip'])
-                            elif select == 'closeness':
-                                rep = sort_closeness(dictreplica, client_location)
-                            elif select == 'dynamic':
-                                rep = sort_dynamic(dictreplica, client_location)
-                            elif select == 'ranking':
-                                rep = sort_ranking(dictreplica, client_location)
-                            else:
-                                rep = sort_random(dictreplica)
-
+                            rep = sort_replicas(dictreplica, client_location, selection=select, default='random')
```

Comment:
same here?

Reply:
no, the redirect endpoint is out of scope.